packages/vscode-extension/src/webview/views/DevicesNotFoundView.tsx

```diff
@@ -97,6 +97,10 @@ function DevicesNotFoundView() {
   }
 
   async function createIOSDevice() {
+    if (!isOSX) {
+      return
+    }
```

Comment:
Just don't call it on unavailable platforms, there's no need to guard against it.